atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -144,11 +144,6 @@ public int schemaMutationTimeoutMillis() {
     }
 
     @Value.Default
-    public int heartbeatTimePeriodMillis() {
```

Comment:
I noticed this config wasn't documented; we should add a section explaining the heartbeat (and also the schema mutation lock more generally)